script.sql

```diff
@@ -1,147 +1,85 @@
-CREATE DATABASE Lab2;
-USE Lab2;
+-- CREATE
 CREATE TABLE Publisher ( 
-id INT,
-Name varchar(60),
- primary key (id)
+        id INT,
+        name VARCHAR(60),
+        PRIMARY KEY (id)
 );
 
 CREATE TABLE Category (
-id INT,
-name varchar(60),
-parentId INT,
-FOREIGN KEY (parentId) REFERENCES Publisher(id) ON DELETE CASCADE ON UPDATE CASCADE,
-primary key (id)
-
-
-/*FOREIGN KEY  parentId REFERENCES */
+        id INT,
+        name varchar(60),
+        parentId INT,
+        PRIMARY KEY (id),
+        FOREIGN KEY (parentId) REFERENCES Category(id)
 );
-CREATE TABLE App  ( 
-id INT,
-name varchar(60),
-publisherId INT,
-categoryId INT,
-/*FOREIGN KEY (publisherId) REFERENCES Publisher(id) ON DELETE CASCADE ON UPDATE CASCADE,
-FOREIGN KEY (categoryId) REFERENCES category(id) ON DELETE CASCADE ON UPDATE CASCADE, */
-description VARCHAR(255)
 
+CREATE TABLE App  ( 
+        id INT,
+        name VARCHAR(60),
+        publisherId INT,
+        categoryId INT,
+        description VARCHAR(255),
+        PRIMARY KEY (id),
+        FOREIGN KEY (publisherId) REFERENCES Publisher(id) 
```

Comment:
Missing foreign key constraints. It's better to move all constraints down to bottom of create statement.

Reply:
Also missing PRIMARY KEY